packages/terra-form/docs/ChoiceField.md

```diff
@@ -0,0 +1,22 @@
+# Choice Field
```

Comment:
I think the main decision was to keep in consistent naming with the textareaField and textField components. I'm not opposed to renaming it radio group though. Perhaps it should be RadioGroupField to indicate that it utilizes a field component as well?